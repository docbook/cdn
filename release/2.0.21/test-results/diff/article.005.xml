<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Unit Test: article.005.xml</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="article" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Unit Test: article.005.xml</h2>
            <h3>The Subtitle</h3>
            <div class="author">
               <h3>
                  <span class="personname">
                     <span class="firstname">Adam</span>
                     <span class="surname">Di Carlo</span>
                  </span>
               </h3>
            </div>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#article.3">Sect1</a>
                     </span>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#article.3.3">Sect2</a>
                           </span>
                        </li>
                     </ul>
                  </li>
                  <li>
                     <span>
                        <a href="#article.4">SecondAbbrev</a>
                     </span>
                  </li>
                  <li>
                     <span>
                        <a href="#article.5">Second1Info</a>
                     </span>
                  </li>
               </ul>
            </div>
         </div>
         <p deltaxml:deltaV2="A">Testing article subtitle, TOC title and sect1info.</p><div deltaxml:deltaV2="B" class="content">
            <p>Testing article subtitle, TOC title and sect1info.</p>
            <section id="article.3" class="sect1">
               <div class="section-titlepage">
                  <h2>Sect1</h2>
               </div>
               <div class="content">
                  <p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p>
                  <section id="article.3.3" class="sect2">
                     <div class="section-titlepage">
                        <h3>Sect2</h3>
                     </div>
                     <div class="content">
                        <p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p>
                     </div>
                  </section>
               </div>
            </section>
            <section id="article.4" class="sect1">
               <div class="section-titlepage">
                  <h2>Second Sect1</h2>
               </div>
               <div class="content">
                  <p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">SecondAbbrev</code>.  Ensure it's using the abbreviated
title in the TOC.</p>
                  <p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p>
               </div>
            </section>
            <section id="article.5" class="sect1">
               <div class="section-titlepage">
                  <h2>Using Sect1Info</h2>
               </div>
               <div class="content">
                  <p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">Second1Info</code>.  Ensure it's using the abbreviated
title in the TOC.</p>
                  <p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p>
               </div>
            </section>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="article.3" class="sect1">
            <div class="section-titlepage">
               <h2>Sect1</h2>
            </div>
            <p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p>
            <section id="article.3.3" class="sect2">
               <div class="section-titlepage">
                  <h3>Sect2</h3>
               </div>
               <p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p>
            </section>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="article.4" class="sect1">
            <div class="section-titlepage">
               <h2>Second Sect1</h2>
            </div>
            <p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">SecondAbbrev</code>.  Ensure it's using the abbreviated
title in the TOC.</p>
            <p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="article.5" class="sect1">
            <div class="section-titlepage">
               <h2>Using Sect1Info</h2>
            </div>
            <p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">Second1Info</code>.  Ensure it's using the abbreviated
title in the TOC.</p>
            <p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p>
         </section>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>