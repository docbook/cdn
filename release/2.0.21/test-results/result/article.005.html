<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for article.005.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: article.005.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">7</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: article.005.xml&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="article"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: article.005.xml&lt;/h2&gt;</span>
            <span class="tag">&lt;h3&gt;The Subtitle&lt;/h3&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Adam&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Di Carlo&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#article.3"</span>&gt;Sect1&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                     <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                        <span class="tag">&lt;li&gt;
                           <span class="tag">&lt;span&gt;
                              <span class="tag">&lt;a <span class="att">href="#article.3.3"</span>&gt;Sect2&lt;/a&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/li&gt;</span>
                     &lt;/ul&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#article.4"</span>&gt;SecondAbbrev&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#article.5"</span>&gt;Second1Info&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff del">&lt;p&gt;Testing article subtitle, TOC title and sect1info.&lt;/p&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;section <span class="att">class="sect1"</span> <span class="att">id="article.3"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;Sect1&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
&lt;/p&gt;</span>
            <span class="tag">&lt;section <span class="att">class="sect2"</span> <span class="att">id="article.3.3"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h3&gt;Sect2&lt;/h3&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;p&gt;
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
&lt;/p&gt;</span>
            &lt;/section&gt;</span>
         &lt;/section&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;section <span class="att">class="sect1"</span> <span class="att">id="article.4"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;Second Sect1&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;
The <span class="tag">&lt;code <span class="att">class="tag-element"</span>&gt;titleabbrev&lt;/code&gt;</span> is set to
<span class="tag">&lt;code <span class="att">class="literal"</span>&gt;SecondAbbrev&lt;/code&gt;</span>.  Ensure it's using the abbreviated
title in the TOC.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
&lt;/p&gt;</span>
         &lt;/section&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;section <span class="att">class="sect1"</span> <span class="att">id="article.5"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;Using Sect1Info&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;
The <span class="tag">&lt;code <span class="att">class="tag-element"</span>&gt;titleabbrev&lt;/code&gt;</span> is set to
<span class="tag">&lt;code <span class="att">class="literal"</span>&gt;Second1Info&lt;/code&gt;</span>.  Ensure it's using the abbreviated
title in the TOC.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
&lt;/p&gt;</span>
         &lt;/section&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Unit Test: article.005.xml&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="article"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Unit Test: article.005.xml&lt;/h2&gt;</span>
            <span class="tag">&lt;h3&gt;The Subtitle&lt;/h3&gt;</span>
            <span class="tag">&lt;div <span class="att">class="author"</span>&gt;
               <span class="tag">&lt;h3&gt;
                  <span class="tag">&lt;span <span class="att">class="personname"</span>&gt;
                     <span class="tag">&lt;span <span class="att">class="firstname"</span>&gt;Adam&lt;/span&gt;</span>
                     <span class="tag">&lt;span <span class="att">class="surname"</span>&gt;Di Carlo&lt;/span&gt;</span>
                  &lt;/span&gt;</span>
               &lt;/h3&gt;</span>
            &lt;/div&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#article.3"</span>&gt;Sect1&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                     <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                        <span class="tag">&lt;li&gt;
                           <span class="tag">&lt;span&gt;
                              <span class="tag">&lt;a <span class="att">href="#article.3.3"</span>&gt;Sect2&lt;/a&gt;</span>
                           &lt;/span&gt;</span>
                        &lt;/li&gt;</span>
                     &lt;/ul&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#article.4"</span>&gt;SecondAbbrev&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#article.5"</span>&gt;Second1Info&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;p&gt;Testing article subtitle, TOC title and sect1info.&lt;/p&gt;</span>
            <span class="tag">&lt;section <span class="att">class="sect1"</span> <span class="att">id="article.3"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Sect1&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
&lt;/p&gt;</span>
                  <span class="tag">&lt;section <span class="att">class="sect2"</span> <span class="att">id="article.3.3"</span>&gt;
                     <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                        <span class="tag">&lt;h3&gt;Sect2&lt;/h3&gt;</span>
                     &lt;/div&gt;</span>
                     <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                        <span class="tag">&lt;p&gt;
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
&lt;/p&gt;</span>
                     &lt;/div&gt;</span>
                  &lt;/section&gt;</span>
               &lt;/div&gt;</span>
            &lt;/section&gt;</span>
            <span class="tag">&lt;section <span class="att">class="sect1"</span> <span class="att">id="article.4"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Second Sect1&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;
The <span class="tag">&lt;code <span class="att">class="tag-element"</span>&gt;titleabbrev&lt;/code&gt;</span> is set to
<span class="tag">&lt;code <span class="att">class="literal"</span>&gt;SecondAbbrev&lt;/code&gt;</span>.  Ensure it's using the abbreviated
title in the TOC.&lt;/p&gt;</span>
                  <span class="tag">&lt;p&gt;
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/section&gt;</span>
            <span class="tag">&lt;section <span class="att">class="sect1"</span> <span class="att">id="article.5"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;Using Sect1Info&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;
The <span class="tag">&lt;code <span class="att">class="tag-element"</span>&gt;titleabbrev&lt;/code&gt;</span> is set to
<span class="tag">&lt;code <span class="att">class="literal"</span>&gt;Second1Info&lt;/code&gt;</span>.  Ensure it's using the abbreviated
title in the TOC.&lt;/p&gt;</span>
                  <span class="tag">&lt;p&gt;
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/section&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="article" class="article"><header class="article-titlepage"><h2>Unit Test: article.005.xml</h2><h3>The Subtitle</h3><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#article.3">Sect1</a></span><ul class="toc"><li><span><a href="#article.3.3">Sect2</a></span></li></ul></li><li><span><a href="#article.4">SecondAbbrev</a></span></li><li><span><a href="#article.5">Second1Info</a></span></li></ul></div></div><p>Testing article subtitle, TOC title and sect1info.</p><section id="article.3" class="sect1"><div class="section-titlepage"><h2>Sect1</h2></div><p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p><section id="article.3.3" class="sect2"><div class="section-titlepage"><h3>Sect2</h3></div><p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p></section></section><section id="article.4" class="sect1"><div class="section-titlepage"><h2>Second Sect1</h2></div><p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">SecondAbbrev</code>.  Ensure it's using the abbreviated
title in the TOC.</p><p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p></section><section id="article.5" class="sect1"><div class="section-titlepage"><h2>Using Sect1Info</h2></div><p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">Second1Info</code>.  Ensure it's using the abbreviated
title in the TOC.</p><p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p></section></article><script src="../../resources/js/prism.js"></script></td><td><article id="article" class="article"><header class="article-titlepage"><h2>Unit Test: article.005.xml</h2><h3>The Subtitle</h3><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#article.3">Sect1</a></span><ul class="toc"><li><span><a href="#article.3.3">Sect2</a></span></li></ul></li><li><span><a href="#article.4">SecondAbbrev</a></span></li><li><span><a href="#article.5">Second1Info</a></span></li></ul></div></div><div class="content"><p>Testing article subtitle, TOC title and sect1info.</p><section id="article.3" class="sect1"><div class="section-titlepage"><h2>Sect1</h2></div><div class="content"><p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p><section id="article.3.3" class="sect2"><div class="section-titlepage"><h3>Sect2</h3></div><div class="content"><p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p></div></section></div></section><section id="article.4" class="sect1"><div class="section-titlepage"><h2>Second Sect1</h2></div><div class="content"><p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">SecondAbbrev</code>.  Ensure it's using the abbreviated
title in the TOC.</p><p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p></div></section><section id="article.5" class="sect1"><div class="section-titlepage"><h2>Using Sect1Info</h2></div><div class="content"><p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">Second1Info</code>.  Ensure it's using the abbreviated
title in the TOC.</p><p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p></div></section></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/article.005.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>