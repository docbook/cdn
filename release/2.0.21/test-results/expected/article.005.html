<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Unit Test: article.005.xml</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="article" class="article"><header class="article-titlepage"><h2>Unit Test: article.005.xml</h2><h3>The Subtitle</h3><div class="author"><h3><span class="personname"><span class="firstname">Adam</span> <span class="surname">Di Carlo</span></span></h3></div></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#article.3">Sect1</a></span><ul class="toc"><li><span><a href="#article.3.3">Sect2</a></span></li></ul></li><li><span><a href="#article.4">SecondAbbrev</a></span></li><li><span><a href="#article.5">Second1Info</a></span></li></ul></div></div><p>Testing article subtitle, TOC title and sect1info.</p><section id="article.3" class="sect1"><div class="section-titlepage"><h2>Sect1</h2></div><p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p><section id="article.3.3" class="sect2"><div class="section-titlepage"><h3>Sect2</h3></div><p>
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
Some text.  Some text.  Some text.  Some text.  Some text.  Some text.
</p></section></section><section id="article.4" class="sect1"><div class="section-titlepage"><h2>Second Sect1</h2></div><p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">SecondAbbrev</code>.  Ensure it's using the abbreviated
title in the TOC.</p><p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p></section><section id="article.5" class="sect1"><div class="section-titlepage"><h2>Using Sect1Info</h2></div><p>
The <code class="tag-element">titleabbrev</code> is set to
<code class="literal">Second1Info</code>.  Ensure it's using the abbreviated
title in the TOC.</p><p>
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.  Foo bar foo bar.
Foo bar foo bar.  Foo bar foo bar.
</p></section></article><script src="../../resources/js/prism.js"></script></body></html>